polaris-core/src/main/java/org/apache/polaris/core/config/RealmConfig.java

```diff
@@ -31,7 +31,9 @@ public interface RealmConfig {
    * @param <T> the type of the configuration value
    * @param configName the name of the configuration key to check
    * @return the current value set for the configuration key, or null if not set
+   * @deprecated Use typed {@link #getConfig(PolarisConfiguration)} instead.
```

Comment:
do we have a version in mind, where were wanna remove it, if we do it would be nice to add it here !